docs/install/lotus/troubleshooting.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Troubleshooting Lotus installation
+description: If you're having issues installing Lotus on your system, try running through some of these troubleshooting steps.
+---
+
+# Troubleshooting
+
+If you're having trouble installing Lotus, try running through some of these troubleshooting steps. If you're still having trouble, feel free to reach out on the `#fil-help` channel in the Filecoin Slack.
+
+## Update your operating system
+
+This sounds simple, but a common reason users aren't able to run Filecoin is because their operating system isn't up to date.
+
+## Check the dependencies
+
+Lotus has several dependencies that must be installed _before_ the software can run. Run the command below for your operating system to make sure everything is installed and up-to-date.
+
+| Operating system | Update command                                                                                                             |
+| ---------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| Ubuntu           | `sudo apt update && sudo apt install mesa-opencl-icd ocl-icd-opencl-dev gcc git bzr jq pkg-config curl build-essential -y` |
+| macOS            | `brew install go bzr jq pkg-config rustup git`                                                                             |
+
```

Comment:
Link back to instructions rather than duplicating dependency-install instructions here in an uncomplete way.

Reply:
Done